ecl/eclccserver/eclccserver.cpp

```diff
@@ -893,6 +893,11 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
         {
             if (!useChildProcesses && !childProcessTimeLimit && !config->hasProp("@workunit"))
             {
+                {
+                    Owned<IWorkUnitFactory> factory = getWorkUnitFactory();
+                    Owned<IWorkUnit> wu = factory->updateWorkUnit(wuid.get());
+                    wu->setContainerizedProcessInfo("EclCCServer", getComponentConfigSP()->queryProp("@name"), queryMyPodName(), nullptr);
```

Comment:
I debated if this is really worth it - before the dali fix will it may be an overhead.

Reply:
well, it's worth it in the sense, that I think this will be become our primary route into logging, without knowing the entry point (pod name) for the eclccserver that ran the job - it's going to be much harder to find the relevant logging.
Admittedly the eclccserver is more static than other components, but still, a updated deployment = new pod etc.